tests/phpunit/test-class-wc-stripe-upe-payment-gateway.php

```diff
@@ -117,6 +117,8 @@ class WC_Stripe_UPE_Payment_Gateway_Test extends WP_UnitTestCase {
 	public function set_up() {
 		parent::set_up();
 
+		update_option( WC_Stripe_Feature_Flags::LPM_ACH_FEATURE_FLAG_NAME, 'yes' );
```

Comment:
Did you double-check that this is working? For Bacs, I had to place it in `set_up_before_class`.

Reply:
I had to do that in tests/phpunit/admin/test-class-wc-rest-stripe-settings-controller.php BTW

Reply:
This seemed to work for me. All the unit tests passed and when I remove this line the `test_get_upe_available_payment_methods` test fails as it's missing ACH. So it seems this is properly setting the option value and tear down properly removing it.